src/bo4e_generator/sqlparser.py

```diff
@@ -365,13 +372,36 @@ def write_many_many_links(links: dict[str, str]) -> str:
     environment = Environment(loader=FileSystemLoader(template_path))
     template = environment.get_template("ManyLinks.jinja2")
     python_code = template.render({"class": links})
-    python_code = format_code(python_code)
+    # python_code = format_code(python_code)
     return python_code
 
 
 def format_code(code: str) -> str:
     """
     perform isort and black on code
     """
+    code = remove_unused_imports(code)
     code = black.format_str(code, mode=black.Mode())
-    return isort.code(code, known_local_folder=["borm"])
+    return isort.code(code)
```

Comment:
Wird `format_code` jetzt überhaupt noch verwendet? `remove_unused_imports` rufst du ja jetzt auch in `generate_bo4e_schemas` auf und mittlerweile sollte das Formatieren auch korrekt funktionieren, meine ich..